.vimrc

```diff
@@ -28,9 +28,9 @@ Plug 'airblade/vim-gitgutter'
 call plug#end()
 
 " 80 characters line
-set colorcolumn=81
-execute "set colorcolumn=" . join(range(81,335), ',')
-highlight ColorColumn ctermbg=Black ctermfg=DarkRed
+" set colorcolumn=81
+" execute "set colorcolumn=" . join(range(81,335), ',')
+" highlight ColorColumn ctermbg=Black ctermfg=DarkRed
```

Comment:
No need to leave this stuff as a comment ("). Just go ahead and drop the lines completely.